samples/04_gis_analysts_data_scientists/chennai_floods_analysis.ipynb

```diff
@@ -790,7 +790,7 @@
    "source": [
```

Comment:
<p><strong>It seems the cleanchennai domain has lapsed and the link no longer works?</strong></p><p> </p><p>To provide emergency assistance, the Tamil Nadu government has set up several flood relief camps in the flood affected areas. They provide food, shelter, and basic necessities to thousands of people displaced by the floods. The locations of the flood relief camps were obtained from <a href="http://cleanchennai.com/floodrelief/2015/12/09/relief-centers-as-on-8-dec-2015/" target="_blank">http://cleanchennai.com/floodrelief/2015/12/09/relief-centers-as-on-8-dec-2015/</a> and <a href="https://ciifoundation.in/Tamil-Nadu-Flood-Relief-2015.php" target="_blank">https://ciifoundation.in/Tamil-Nadu-Flood-Relief-2015.php</a> and were published to the GIS as a layer that is visualized below:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>